src/ui/public/notify/notifier.js

```diff
@@ -111,6 +115,18 @@ function add(notif, cb) {
     return notif.timerId ? true : false;
   };
 
+  // decorate the notification with helper functions for the template
+  notif.getButtonClass = () => `btn-${notif.type}`;
```

Comment:
I think we should apply the same patterns to any situation where we're dynamically building a class name based on the `notif.type`. Instead of that, I think we should explicitly declare a relationship between specific types and specific class names. For example:

```javascript
notif.getButtonClass = () => {
  const typeToButtonClassMap = {
    error: 'btn-danger',
    warning: 'btn-warning',
    info: 'btn-info',
  };
  return typeToButtonClassMap[notif.type];
};
```

Since this map is used in a couple places, we could also extract it to the enclosing, shared scope.